02.calendar/calendar.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+# コマンドラインオプション設定(デフォルト値は今日の年と月)
+params =ARGV.getopts("m:", "y:", "m:#{Date.today.mon}", "y:#{Date.today.year}")
+input_month = params["m"].to_i
+input_year = params["y"].to_i
+
+# 月の最初の日にちを取得
+start_date = Date.new(input_year, input_month, 1)
+
+# 月の最終日を取得
+end_date = Date.new(start_date.year, start_date.mon, -1)
+
+# 月と年を取得
+header = "#{start_date.mon}月 #{start_date.year}"
```

Comment:
```suggestion
header = "#{start_date.mon}月 #{start_date.year}"
puts header.center(20)
```

変数の寿命をなるべく短くした方がいいです

https://bootcamp.fjord.jp/pages/311

Reply:
修正しました！
0e34fc5a01c2dd12bb1104c5fcc09fd8640a227c

Reply:
https://github.com/jun-kondo/ruby-practices/pull/2#discussion_r1460693842 でコメントしたようにここもyearとmonthを使う方がベターです。

Reply:
こちら修正しました!!
eb79f10b5b0382bedb8755d6d44dd15043906d08